packages/peregrine/lib/talons/Header/useStoreSwitcher.js

```diff
@@ -12,13 +12,15 @@ const mapAvailableOptions = (config, stores) => {
     const { code: configCode } = config;
 
     return stores.reduce((map, store) => {
+        console.warn(store);
```

Comment:
`console.warn()` to remove if not needed.

Reply:
oops, thanks for catching that, removed.